### PR TITLE
added get loan-storage\loans test

### PR DESCRIPTION
**Pull request by Roman-Fedynyshyn**

### CODE REVIEW

File: workflows-scripts/circulation/loans/loan-storage/jmeter-supported-data/UserID.txt [hunk 1]
```diff
@@ -0,0 +1,5420 @@
+e1f39836-715a-4530-a03b-3cb3af0299af
```
Comment (mtraneis) [inline workflows-scripts/circulation/loans/loan-storage/jmeter-supported-data/UserID.txt:1]: Please rename this file to user_ids.csv to follow other files' convention.

File: workflows-scripts/circulation/loans/loan-storage/GET_loan-storage_loans.jmx [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+            </elementProp>
+          </collectionProp>
+        </HeaderManager>
+        <hashTree/>
+        <HTTPSamplerProxy guiclass="HttpTestSampleGui" testclass="HTTPSamplerProxy" testname="GET /loan-storage/loans?(userId=={UserID} and status.name==Open)" enabled="true">
+          <elementProp name="HTTPsampler.Arguments" elementType="Arguments" guiclass="HTTPArgumentsPanel" testclass="Arguments" enabled="true">
+            <collectionProp name="Arguments.arguments">
+              <elementProp name="query" elementType="HTTPArgument">
+                <boolProp name="HTTPArgument.always_encode">true</boolProp>
+                <stringProp name="Argument.value">(userId==${UserID} and status.name==Open)</stringProp>
```
Comment (mtraneis) [inline workflows-scripts/circulation/loans/loan-storage/GET_loan-storage_loans.jmx:215]: Please change **status.name<>Closed**  as in the original request.
```suggestion
                <stringProp name="Argument.value">(userId==${UserID} and status.name==Closed)</stringProp>
```

File: workflows-scripts/circulation/loans/loan-storage/GET_loan-storage_loans.jmx [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+                <stringProp name="Argument.metadata">=</stringProp>
+                <boolProp name="HTTPArgument.use_equals">true</boolProp>
+                <stringProp name="Argument.name">query</stringProp>
+              </elementProp>
+              <elementProp name="limit" elementType="HTTPArgument">
+                <boolProp name="HTTPArgument.always_encode">false</boolProp>
+                <stringProp name="Argument.value">2147483647</stringProp>
```
Comment (mtraneis) [inline workflows-scripts/circulation/loans/loan-storage/GET_loan-storage_loans.jmx:222]: limit should be 1.
```suggestion
                <stringProp name="Argument.value">1</stringProp>
```

File: workflows-scripts/circulation/loans/loan-storage/GET_loan-storage_loans.jmx [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+                <boolProp name="HTTPArgument.use_equals">true</boolProp>
+                <stringProp name="Argument.name">limit</stringProp>
+              </elementProp>
+            </collectionProp>
+          </elementProp>
+          <stringProp name="HTTPSampler.domain"></stringProp>
+          <stringProp name="HTTPSampler.port"></stringProp>
+          <stringProp name="HTTPSampler.protocol"></stringProp>
+          <stringProp name="HTTPSampler.contentEncoding"></stringProp>
+          <stringProp name="HTTPSampler.path">/loan-storage/loans?</stringProp>
```
Comment (mtraneis) [inline workflows-scripts/circulation/loans/loan-storage/GET_loan-storage_loans.jmx:233]: I don't think you need the / and ? 
```suggestion
          <stringProp name="HTTPSampler.path">loan-storage/loans</stringProp>
```

File: workflows-scripts/circulation/loans/loan-storage/GET_loan-storage_loans.jmx [hunk 5]
```diff
@@ -0,0 +1,305 @@
...
+          <stringProp name="HTTPSampler.method">GET</stringProp>
+          <boolProp name="HTTPSampler.follow_redirects">true</boolProp>
+          <boolProp name="HTTPSampler.auto_redirects">false</boolProp>
+          <boolProp name="HTTPSampler.use_keepalive">true</boolProp>
+          <boolProp name="HTTPSampler.DO_MULTIPART_POST">false</boolProp>
+          <stringProp name="HTTPSampler.embedded_url_re"></stringProp>
+          <stringProp name="HTTPSampler.connect_timeout"></stringProp>
+          <stringProp name="HTTPSampler.response_timeout"></stringProp>
+          <stringProp name="TestPlan.comments">userId==aa1aa31a-ef6c-4e7f-8ac9-a7cd1d23010e and </stringProp>
```
Comment (mtraneis) [inline workflows-scripts/circulation/loans/loan-storage/GET_loan-storage_loans.jmx:242]: May want to revise the comment to something more intuitive

File: workflows-scripts/circulation/loans/loan-storage/GET_loan-storage_loans.jmx [hunk 6]
```diff
@@ -0,0 +1,305 @@
...
+        <stringProp name="ThreadGroup.duration">${duration}</stringProp>
+        <stringProp name="ThreadGroup.delay"></stringProp>
+        <stringProp name="TestPlan.comments">${global_vusers}          ${global_rampup}</stringProp>
+        <boolProp name="ThreadGroup.same_user_on_next_iteration">true</boolProp>
+      </ThreadGroup>
+      <hashTree>
+        <CSVDataSet guiclass="TestBeanGUI" testclass="CSVDataSet" testname="CSV Data Set Config - IDs" enabled="true">
+          <stringProp name="delimiter">,</stringProp>
+          <stringProp name="fileEncoding">UTF-8</stringProp>
+          <stringProp name="filename">${global_BaseDir}/jmeter-supported-data/UserID.txt</stringProp>
```
Comment (mtraneis) [inline workflows-scripts/circulation/loans/loan-storage/GET_loan-storage_loans.jmx:180]: Need to update "/UserID.txt" once you've renamed the file

**Review by mtraneis — COMMENTED** [review]
There are no major issues so it could be used with minor changes (see comments).
Other issues:
1. Please remove .DS_Store files. These are MacOS files, and not needed.
2. Move circulation/loans/loan-storage directory to **/api-scripts** instead.
3. Create a README page, see other READMEs for example.

File: workflows-scripts/circulation/loans/loan-storage/jmeter-supported-data/UserID.txt (hunk 1 above)
Comment (Roman-Fedynyshyn) [reply workflows-scripts/circulation/loans/loan-storage/jmeter-supported-data/UserID.txt:1]: done

File: api-scripts/loan-storage/README.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+##### Backend:
+- mod-circulation-18.0.9
+- mod-circulation-storage-11.0.0
+- mod-authtoken-2.4.0
+- mod-permissions-5.9.0
+- okapi-2.38.0
+##### Frontend:
+- folio_circulation-2.0.0
+
+- FOLIO release: HONEYSOUCKLE
```
Comment (mtraneis) [inline api-scripts/loan-storage/README.md:18]: Please add a comment on how to run the test, specifically where the user IDs are obtained from?
1) Restore the database
2) Run the circ data load script
3) anything else..?

**Pull request merged**